### DISCUSSION TITLE
Microservices with AWS SQS handling events

### DISCUSSION THREAD

**Discussion by bmbBAMBUS** [spatie/laravel-event-sourcing#322] [Q&A]
Hi, 
I consider how to use your package with events from another microservice dispatched via sns & sqs.

I receive messages from another microservices which contains uuid of another aggregate which does not belong to my microservice. 
Should I make new "virtual" and save that events and make proper reactors with actions for another aggregates? 
Or should I try to find uuid of my aggregates via projections related to these foregin aggregates?

What is the best/preferred way to save them and handle?

**Comment by brendt** [comment]
If you want your microservice to be as detached as possible, it would be a good idea to trigger new internal events that focus on your business logic. That's probably what I'd do, unless you've got a good reason otherwise.
